src/main/flight/pid.c

```diff
@@ -93,6 +93,21 @@ void pidResetErrorGyro(void)
 
 const angle_index_t rcAliasToAngleIndexMap[] = { AI_ROLL, AI_PITCH };
 
+
+// Low Pass filter for PID controllers
+void pt1_element(int axis, float *delta, uint8_t f_cut, int type) {
+
+	float dT = (float)cycleTime * 0.000001f;
+
+    float    RC;
+    static float    lastDelta[][3] = {{ 0.0f, 0.0f, 0.0f }};
+
```

Comment:
@ledvinap My gcc linux compiled doesn't mind that. Try for yourself. I can add 3 to make it sure

@henn1001 So you are confirming that it works? That's nice to hear. I will check your branch.
I only want to do a better approach of gyro filtering so that's why I didn't include it here.